packages/web-app-files/src/components/AppBar/AppBar.vue

```diff
@@ -192,10 +191,12 @@ export default {
     ...mapState('Files', ['areHiddenFilesShown']),
 
     mimetypesAllowedForCreation() {
-      if (!get(this, 'mimeTypes', []).length) {
+      // we can't use `mapGetters` here because the External app doesn't exist in all deployments
+      const mimeTypes = this.$store.getters['External/mimeTypes']
+      if (!mimeTypes || mimeTypes !== {}) {
```

Comment:
Isn't this *always* true as `{}` always instantiates a new object which is never the same as `mimeTypes`?

Reply:
Great catch, apparently one should go for sth like this https://flaviocopes.com/javascript-check-empty-object/

Turns out we  (=past me) were initializing the `mimeTypes` into an empty object and then overwriting it as an array, fixing that by turning it into an array from the get-go so we can drop this line :)